# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/two-xing-header-no-content-length.mp3
# Commit: f33e1ce6dcf1
# Full Hash: f33e1ce6dcf19e47952f3963c8e970c7c380acd2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-20 21:19:23
# Regressor Bug: 1843003
# File Overlap Count: 2
# Description:
#   Bug 1843003 - Test that typical mp3 broadcast work well. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183737
# ==============================================================================

diff -r 763b34de3c14 -r f33e1ce6dcf1 dom/media/test/two-xing-header-no-content-length.mp3
Binary file dom/media/test/two-xing-header-no-content-length.mp3 has changed